mongodb/README.md

```diff
@@ -27,11 +27,11 @@ This plugin creates the following helper files:
 
 * **devbox.d/mongodb/mongod.conf** - MongoDB configuration file
 * **.devbox/virtenv/mongodb/data** - empty directory for holding your mongodb database
-* **.devbox/virtenv/rabbitmq/process-compose.yaml** - Defines the process to start the MongoDB server
+* **.devbox/virtenv/mongodb/process-compose.yaml** - Defines the process to start the MongoDB server
 
 ## Environment Variables
 
 This plugin sets the following environment variables:
 
 * **MONGODB_CONFIG** = {{.DevboxDir}}/mongod.conf
-* **MONGODB_DATA** = {{.Virtenv}}/data
\ No newline at end of file
+* **MONGODB_DATA** = {{.Virtenv}}/data
```

Comment:
Don't know why we have line showing in the diff. Edit and PR was all done with Github Web UI.